pom.xml

```diff
@@ -194,57 +128,8 @@
 						<appLocation>${project.artifactId}.war</appLocation>
 					</bootstrapProperties>
 				</configuration>
-				<executions>
-					<execution>
-						<id>install-liberty</id>
-						<phase>prepare-package</phase>
-						<goals>
-							<goal>install-server</goal>
-						</goals>
-					</execution>
-					<execution>
-						<id>package-app</id>
-						<phase>package</phase>
-						<goals>
-							<goal>package-server</goal>
-						</goals>
-					</execution>
-				</executions>
 			</plugin>
 			<!-- Plugin to run functional tests -->
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-failsafe-plugin</artifactId>
-				<version>2.18.1</version>
-				<executions>
-					<execution>
-						<phase>integration-test</phase>
-						<id>integration-test</id>
-						<goals>
-							<goal>integration-test</goal>
-						</goals>
-						<configuration>
-							<includes>
-								<include>**/it/**</include>
```

Comment:
With this `<include>**/it/**</include>` removed, you IT test isn't running by default.  If you look at the failsafe plugin documentation (http://maven.apache.org/surefire/maven-failsafe-plugin/examples/inclusion-exclusion.html), it will auto run 

    "**/IT*.java" - includes all of its subdirectories and all Java filenames that start with "IT".
    "**/*IT.java" - includes all of its subdirectories and all Java filenames that end with "IT".
    "**/*ITCase.java" - includes all of its subdirectories and all Java filenames that end with "ITCase".
So, to keep this simple just rename the test in the /it/ folder EndpointIT.java